docs/feature_combo.md

```diff
@@ -67,31 +86,100 @@ void process_combo_event(uint8_t combo_index, bool pressed) {
         tap_code16(LCTL(KC_V));
       }
       break;
+    case SD_LAYER:
+      /* until more advanced keycodes are supported, layer changing combos need to be handled manually. */
+      if (pressed) {
+        layer_on(_LAYER);
+      } else {
+        layer_off(_LAYER);
+      }
+      break;
   }
 }
 ```
 
-This will send Ctrl+C if you hit Z and C, and Ctrl+V if you hit X and V.  But you could change this to do stuff like change layers, play sounds, or change settings.
-
-## Additional Configuration
-
-If you're using long combos, or even longer combos, you may run into issues with this, as the structure may not be large enough to accommodate what you're doing.
-
-In this case, you can add either `#define EXTRA_LONG_COMBOS` or `#define EXTRA_EXTRA_LONG_COMBOS` in your `config.h` file.
+This will send Ctrl+C if you hit Z and C, Ctrl+V if you hit X and V, and changel layer to `_LAYER` when you hold down S and D.  But you could change this to do stuff like play sounds or change settings.
 
-You may also be able to enable action keys by defining `COMBO_ALLOW_ACTION_KEYS`.
+# Advanced Configuration
+These configuration settings can be set in your keymaps `config.h`. 
```

Comment:
« in your keymap's `config.h`. » or « in your `config.h` file. » would be better.